smoke_tests/run-smoke-test.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+ENV='local'
+LSJ_SECRET='terry'
+MSJ_SECRET='bob'
+LSJ_URL='http://localhost:3000'
+MSJ_URL='http://localhost:3000'
+CYPRESS_EXE=./node_modules/.bin/cypress
```

Comment:
Defaults to values required when running smoke tests locally - not quite "check out and run" but as close as possible.